Library/Homebrew/extend/os/mac/keg_relocate.rb

```diff
@@ -38,6 +38,23 @@ def relocate_dynamic_linkage(relocation)
     end
   end
 
+  # Ensures that an ad-hoc signature is present on all binaries,
+  # using the OS's builtin codesign tool.
+  # This is probably temporary - while codesigning is required on
```

Comment:
If modifying binaries when pouring bottles via `ruby-macho` is still a thing, then this won't be a temporary workaround, but a necessary step.

Reply:
Yes, it is so: good to know 👍🏻

Reply:
That's a very good point. I'll update this comment to note that.